packages/react-native/src/primitives/Button/Button.tsx

```diff
@@ -7,28 +7,37 @@ import {
   ViewStyle,
 } from 'react-native';
 
-import { styles } from './styles';
+import { useTheme } from '../../theme';
+import { getThemedStyles } from './styles';
 import { ButtonProps } from './types';
 
 export default function Button({
+  accessibilityRole = 'button',
   children,
```

Comment:
Thanks for adding default accessibility role! We should also support `variants` and `size`  for the button.